src/scenes/Projects/Overview/ProjectOverview.tsx

```diff
@@ -514,6 +521,44 @@ export const ProjectOverview: FC = () => {
             skeletonCount={0}
             renderSkeleton={null}
           />
+
+          <Grid container spacing={2} alignItems="center">
+            <Grid item>
+              <Typography variant="h3">Comments</Typography>
+            </Grid>
+            <Grid item>
+              <Tooltip title={`Add Comment`}>
+                <Fab
+                  color="error"
+                  aria-label={`Add Comment`}
+                  onClick={toggleCommentField}
+                >
+                  <Add />
+                </Fab>
+              </Tooltip>
+            </Grid>
+          </Grid>
+          {showCommentField && (
+            <CreatePostForm
+              parentId={projectId}
+              onSubmit={() => {
+                toggleCommentField();
+
+                // FIXME: need to figure out how to do list cache-updates for nested lists
+                refetch().catch((err) => console.log('refetch error', err));
```

Comment:
Like this https://github.com/SeedCompany/cord-field/blob/7a789fba4da5b209e25e838fc5f8bbb8f83a2deb/src/scenes/Projects/Members/Create/CreateProjectMember.tsx#L53-L58